app/javascript/components/collections/landing/SearchResults.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import '../Collection.scss';
+import SearchResultsCard from './SearchResultsCard';
+
+const SearchResults = props => {
+  return (
+    <ul className="search-within-search-results">
+      {props.documents.map((doc, index) => (
+        <SearchResultsCard
+          key={index}
```

Comment:
Same here, wherever possible we should not use `index` as a value for `key`.  If there's some other unique identifier to grab onto...